stacks-common/src/util/uint.rs

```diff
@@ -141,6 +142,68 @@ macro_rules! construct_uint {
                 }
                 ret
             }
+
+            /// as big-endian byte array
+            pub fn to_u8_slice_be(&self) -> [u8; $n_words * 8] {
+                let mut ret = [0u8; $n_words * 8];
+                for i in 0..$n_words {
+                    let bytes = self.0[i].to_le_bytes();
+                    for j in 0..bytes.len() {
+                        ret[$n_words * 8 - 1 - (i * 8 + j)] = bytes[j];
+                    }
```

Comment:
I think using `to_be_bytes()` makes this a little easier to follow

```suggestion
                    let word_end = $n_words * 8 - (i * 8);
                    let word_start = word_end - 8;
                    ret[word_start..word_end].copy_from_slice(&self.0[i].to_be_bytes());
```

Reply:
Will do